### ISSUE TITLE
Implement Debug Mode

### ISSUE THREAD

**Issue by Schlabbi** [Schlabbi/docker-notify#11]
Implement a debug mode so a verification of the webhooks and mail server connection is easier to do.

Could work like this: Set a flag in the config, the container will then try to send a test mail and hit all the webhooks.

**Comment by mbierman** [comment]
Can there be more than one mail instance? if so then it should hit all webhooks and all mail instances, right?

**Comment by Schlabbi** [comment]
Yes, there can be more than one mail instance. So in this case it would just send a test mail for all instances.

**Comment by mbierman** [comment]
any update on this? :)
